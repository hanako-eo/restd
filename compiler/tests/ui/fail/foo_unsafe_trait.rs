#[compiler::item("foo_unsafe")]
trait Foo {}

fn main() {}
