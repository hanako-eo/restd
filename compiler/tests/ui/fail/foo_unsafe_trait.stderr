error: `foo_unsafe` item must be an unsafe trait
 --> tests/ui/fail/foo_unsafe_trait.rs:2:1
  |
2 | trait Foo {}
  | ^^^^^^^^^^^^
